src/authentication.ts

```diff
@@ -69,30 +81,30 @@ export const fetchTokens = (config: TInternalConfig): Promise<TTokenResponse> =>
     throw Error("Can't get tokens without the CodeVerifier. \nHas authentication taken place?")
   }
 
-  const formData = new FormData()
-  formData.append('grant_type', 'authorization_code')
-  formData.append('code', authCode)
-  formData.append('scope', config.scope)
-  formData.append('client_id', config.clientId)
-  formData.append('redirect_uri', config.redirectUri)
-  formData.append('code_verifier', codeVerifier)
-
-  return postWithFormData(config.tokenEndpoint, formData)
+  const tokenRequest: TTokenRequest = {
+    grant_type: 'authorization_code',
+    code: authCode,
+    scope: config.scope,
+    client_id: config.clientId,
+    redirect_uri: config.redirectUri,
+    code_verifier: codeVerifier,
+  }
+  return postWithFormData(config.tokenEndpoint, tokenRequest)
 }
 
 export const fetchWithRefreshToken = (props: {
   config: TInternalConfig
   refreshToken: string
 }): Promise<TTokenResponse> => {
   const { config, refreshToken } = props
-  const formData = new FormData()
-  formData.append('grant_type', 'refresh_token')
-  formData.append('refresh_token', refreshToken)
-  formData.append('scope', config.scope)
-  formData.append('client_id', config.clientId)
-  formData.append('redirect_uri', config.redirectUri)
-
-  return postWithFormData(config.tokenEndpoint, formData)
+  const tokenRequest: TTokenRequest = {
```

Comment:
```suggestion
  const tokenRequest: TTokenRequestForRefresh = {
```
Same here. Should use the explicit type